packages/twenty-server/src/engine/core-modules/auth/auth.module.ts

```diff
@@ -96,6 +100,10 @@ import { JwtAuthStrategy } from './strategies/jwt.auth.strategy';
     OnboardingModule,
     WorkspaceDataSourceModule,
     ConnectedAccountModule,
+    MessagingGmailDriverModule,
```

Comment:
this seems wrong, the auth module should not take dependency on driver, this is going the wrong way

Reply:
I understand why you did that now.

What I think we should do:
- think message-folder sync as a sub module of messaging (introduce a message-folder folder under messaging)
- driver should also be within this folder and implement the same interface
- call messageFolderSync when we start fetching message-list (no need for an additional cron + sync command + no need to import in auth)